packages/mdc-checkbox/foundation.ts

```diff
@@ -98,7 +101,19 @@ export class MDCCheckboxFoundation extends MDCFoundation<MDCCheckboxAdapter> {
   }
 
   /**
-   * Handles the change event for the checkbox
+   * Handles the click event for the checkbox
+   * added for IE browser to fix compatibility issue b/144588184
```

Comment:
Remove this comment about IE and add it inside the handler (see my comment below). Also include the link to the bug on GitHub